examples/tutorials/Part 06 - Federated Learning on MNIST using a CNN.ipynb

```diff
@@ -112,7 +112,10 @@
     "\n",
     "device = torch.device(\"cuda\" if use_cuda else \"cpu\")\n",
     "\n",
-    "kwargs = {'num_workers': 1, 'pin_memory': True} if use_cuda else {}"
+    "kwargs = {'num_workers': 1, 'pin_memory': True} if use_cuda else {}\n",
+    "\n",
+    "torch.set_default_tensor_type(torch.cuda.FloatTensor)\n",
+    "torch.multiprocessing.set_start_method('spawn')"
```

Comment:
If it's only for cuda I'd make it optional base on the boolean `use_cuda`

Reply:
Yes, I just tested on CPU. These errors are only for a cuda device.